src/pages/[id].jsx

```diff
@@ -0,0 +1,10 @@
+import { useRouter } from 'next/router';
+
+export default function Details() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  return (
+    <h1>{ id }</h1>
```

Comment:
In the definition of done, the header needs to show up on all pages, so that would include this and the 404 page. We'll want to use the `PageTitle` component as well.